src/styles/footer.scss

```diff
@@ -0,0 +1,150 @@
+.footer {
+  background-color: $footer-background;
+  color: $footer-text-collor;
+  padding: 30px 20px 40px;
+
+  @include tablet {
+    padding: 30px 27px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    column-gap: 30px;
+  }
+
+  @include desktop {
+    column-gap: 120px;
+    padding: 60px 50px;
+    max-width: 1280px;
+  }
+
+  @include big-desktop {
+    padding: 60px 130px;
+    max-width: 1440px;
+  }
+
+  &__menu {
+    display: flex;
+    flex-direction: column;
+    gap: 11px;
+    font-weight: 500;
+    font-size: 18px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 50px;
+    }
+
+    @include desktop {
+      gap: 40px;
+      position: relative;
```

Comment:
Avoid using absolute positioning for the menu items, it may cause issues with different screen sizes. Consider using flexbox or grid for positioning.